.changeset/spicy-fireants-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'playroom': minor
+---
+
+Add keybinding for copying Playroom link to clipboard with <kbd>⌘</kbd> + <kbd>⇧</kbd> + <kbd>C</kbd> (or, on Windows, <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>).
```

Comment:
Should the up arrow just be shift instead? It's a bit small, though I guess that's font-dependent.
```suggestion
Add keybinding for copying Playroom link to clipboard with <kbd>⌘</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd> (or, on Windows, <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>C</kbd>).
```

Reply:
This is MacOS-ism, not worth changing.